content/guide/core/platform.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Platform
+---
+
+This class allows you to access information about the current device such as screen resolution,operating system, model, manufacturer,etc.
+
+
+## Using Platform
+
+The API of the Platform module is accessible through the following imports:
+
+```ts
+import { Device, Screen } from '@nativescript/core'
+```
+
+### Getting a device type
+
+To check if a device is a phone or tablet, use the [deviceType](#devicetype) property.
+
+```ts
+const deviceType: string = Device.deviceType
+```
+
+### Getting a device manufacturer
+
+A device's manufacturer get be accessed using the [manufacturer](#manufacturer) property.
+
+```ts
+const manufacturer: string = Device.manufacturer
+```
+
+### Getting the model of a device
+```ts
+const model: string = Device.model
+```
+
+ ### Checking a device operating system name
+ To get the device operating system name, use the [os](#os) property:
+
+ ```ts
+ const os: string = Device.os
+ ```
+
+### Getting a device operating system version
+
+To access the operating system version of the device, use the [osVersion](#osversion) property:
+
+```ts
+const osVersion: string = Device.osVersion
+```
+
+### Getting the device region
+```ts
+const region: string = Device.region
+```
+
+### Getting the device's sdk version
+
+Use the [sdkVersion](#sdkversion) property to get the
+SDK version:
+```ts
+const sdkVersion: string = Device.sdkVersion
+```
+
+### Getting the device uuid
+To access the device's universally unique id(uuid), use the [uuid](#uuid) property:
+
+```ts
+const uuid: string = Device.uuid
+```
```

Comment:
This section is just reiteration of what all we have said in the docs:Device #52 

Can you please tell why is this repeated here as well?

Reply:
Likely not needed, we will look at removing this.

Reply:
Sure @NathanWalker 
Please let me know if anything is needed